site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -158,6 +158,34 @@ chmod o+r gameserver.yaml
 docker run --network=host --rm -v $(pwd)/gameserver.yaml:/tmp/gameserver.yaml us-docker.pkg.dev/agones-images/release/agones-sdk:{{<release-version>}} --local -f /tmp/gameserver.yaml
 ```
 
+If you would like to run the ClientSDK and your game server container together with Docker Compose create a `docker-compose.yaml` file.
+
+```yaml
+version: '3'
+services:
+  gameserver:
+    build: . # <location to your image here>
+    ports:
+      - "127.0.0.1:7777:7777/udp"
+
+  sdk-server:
+    image: "us-docker.pkg.dev/agones-images/release/agones-sdk:1.34.0"
+    command: --local -f /gs_config
+    network_mode: service:gameserver
```

Comment:
Wondering if we want an extra comment here explaining that this is the shared network?

Reply:
It's a good idea. I added a comment in the code block pointing this out: `# <shared network between sdk and game server>`